packages/app/src/clusterClient.tsx

```diff
@@ -1,6 +1,34 @@
+import React from "react";
+import { StatusAborted, StatusError, StatusOK } from "@backstage/core-components";
 import { ConfigApi } from "@backstage/core-plugin-api";
 import { ClusterDetails } from "@internal/plugin-cluster-status-backend";
 
+
+const clusterApiFetchCall = (
+  configApi: ConfigApi,
+  params: string
+): Promise<any> => {
+  const backendUrl = configApi.getString('backend.baseUrl');
+  const jsonResponse = fetch(`${backendUrl}/api/cluster-status/status${params}`)
+    .then(r => r.json())
+  return jsonResponse;
+}
+
+export const getStatusElement = (cluster: ClusterDetails) => {
+  let status = <StatusAborted />
```

Comment:
Why do you define a `status` variable instead of returning it directly?

```jsx
const getStatusElement = (...) => {
if (available) {
  return <StatusOK />
} else if (down) {
  return <StatusError />
} else {
  return <StatusAborted />
}
}
```

or even as a conditional render: 
```jsx
const getStatusElement = (...) => (
 <>
  {available ? <StatusOk /> : (down ? <StatusError /> : <StatusAborted />)}
 </>
```

Reply:
Dunno why I didn't think of that tbh.